weekly/2024/week-20.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Week 20 in Packit
+date: 2024-05-20
+authors: mmassari
+tags:
+  - 2024-May
+  - 2024
+  - May
+---
+
+## Week 20 (May 14th – May 20th)
+
+- When running `dist-git init` command from CLI, you can pass a command to specify the git URL of the project. ([packit#2308](https://github.com/packit/packit/pull/2308))
```

Comment:
I forgot to put something in https://github.com/packit/dashboard/pull/408 that could be worth adding, something like: 
```suggestion
- When running `dist-git init` command from CLI, you can pass a command to specify the git URL of the project. ([packit#2308](https://github.com/packit/packit/pull/2308))
- We have fixed the incorrect displaying of the time it took for the jobs to run on dashboard. ([dashboard#408](https://github.com/packit/dashboard/pull/408))
```